examples/probabilistic_pca.py

```diff
@@ -8,112 +8,59 @@
 This example shows how to fit feature count data to probabilistic PCA
 model [1].
 
-There are three flavors of probabilistic PCA model that are available:
+There are two flavors of probabilistic PCA model that are available:
```

Comment:
Maybe write a word in the docstring about the pros and cons of the two approaches.